bin/gendocs

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash -e
 
+# Automatic document generation, validation and helpers
+#
+# bin/gendocs ...
+# 
+# <unspecified> (default)
```

Comment:
I don't get this... what's <unspecified> and default?  This comment is more confusing than helpful!

Is "check" and "feature_bucket" the argument to bin/gendocs?  If so, then there should be a placeholder of some kind on line 5 (e.g. "bin/gendocs [cmd]") and then use that to indicate that's what these fields mean... ?